OWNERS_ALIASES

```diff
@@ -240,3 +240,37 @@ aliases:
     - frapposelli
     - yastij
     - randomvariable
+  # end sigs.k8s.io/cluster-api-provider-vsphere/OWNERS_ALIASES
+
+  # mostly copied from sigs.k8s.io/controller-runtime/OWNERS_ALIASES
+  controller-runtime-admins:
+  - directxman12
+  - mengqiy
+
+  controller-runtime-maintainers:
+  - vincepri
+
+  controller-runtime-approvers:
+  - gerred
+
+  controller-runtime-reviewers:
+  - alvaroaleman
+  - alenkacz
+  - vincepri
+  - alexeldeib
+
+  controller-runtime-emeritus-maintainers:
+  - pwittrock
+
+  # mostly copied from sigs.k8s.io/kubebuilder/OWNERS_ALIASES
+  kubebuilder-admins:
+  - mengqiy
+  - directxman12
+
+  kubebuilder-reviewers:
+  - alexeldeib
```

Comment:
I added myself because I added/help maintain these jobs, even though I'm not a kubebuilder reviewer. Also this list is small :/

Reply:
that's fine.  You do a lot of work on this stuff :-)